python/ray/projects/projects.py

```diff
@@ -50,6 +50,44 @@ def working_directory(self):
         directory = os.path.join("~", self.config["name"], "")
         return directory
 
+    def make_argument_parser(self, name, params, wildcards):
```

Comment:
It doesn't look like `self` is getting used in this function, should we move it outside of the class definition?

Reply:
done